Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js

```diff
@@ -2122,6 +2160,12 @@ WI.DOMTreeElement.BreakpointStatus = {
     DisabledBreakpoint: Symbol("disabled-breakpoint"),
 };
 
+WI.DOMTreeElement.Badge = {
+    Flex: "flex",
+    Grid: "grid",
+};
+WI.settings.shownDOMTreeBadges = new WI.Setting("shown-dom-tree-badges", [WI.DOMTreeElement.Badge.Flex, WI.DOMTreeElement.Badge.Grid]);
```

Comment:
`shown` feels like the wrong adjective here. It make me think that it means actually shown to the user, which won't always be true on every page. Maybe `enabledDOMTreeBadgeTypes`?